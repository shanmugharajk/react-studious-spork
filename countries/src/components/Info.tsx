import React from "react";

interface IProps {
  children: React.ReactNode;
}

export const Info: React.FunctionComponent<IProps> = ({ children }) => (
  <div className="mt-5 p-5 bg-gray-50 border border-gray-300 rounded-lg">
    <span className="text-gray-500">{children}</span>
  </div>
);
