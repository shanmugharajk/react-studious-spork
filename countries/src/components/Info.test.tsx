import { render } from "@testing-library/react";

import { Info } from "./Info";

describe("Info", () => {
  it("should render Info", () => {
    const { container } = render(<Info>Info</Info>);

    expect(container).toMatchSnapshot();
  });
});
