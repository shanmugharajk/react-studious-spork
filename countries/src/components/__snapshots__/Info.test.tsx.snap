// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info should render Info 1`] = `
<div>
  <div
    class="mt-5 p-5 bg-gray-50 border border-gray-300 rounded-lg"
  >
    <span
      class="text-gray-500"
    >
      Info
    </span>
  </div>
</div>
`;
